endpointmanager/scripts/populatenpiorgs.go

```diff
@@ -0,0 +1,423 @@
+package main
```

Comment:
my thoughts on how this should be organized/where it should go.

i think everything **but** the `main` function should be included in `pkg/nppesquerier` (or similar). The "querier" piece is for later, but this is all related to that and getting the info into the database.

i think the `main` should go into `cmd/npicsvparser` (or something like that).

i think the test data should go into `pkg/nppesquerier/testdata` (go and govendor know to ignore testdata folders, which is nice).